src/main/webapp/WEB-INF/jsp/transfer.jsp

```diff
@@ -8,45 +8,52 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
 <html>
 <head>
-<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
```

Comment:
Our schema has character encoding as UTF-8 is there any reason to change to ISO-8859-1.